src/core/Table/Table.tsx

```diff
@@ -174,9 +174,9 @@ export const Table = <
 
   const defaultColumn = React.useMemo(
     () => ({
-      maxWidth: 0,
-      minWidth: 0,
-      width: 0,
+      maxWidth: undefined,
```

Comment:
Maybe add a comment why we did this because we will forget 😀

Reply:
Done